docs/circuit-test.md

```diff
@@ -44,6 +44,49 @@ View a history of completed circuit tests, including the provider the call route
 + **Filters** pop out on the right allowing you to filter some of your data.
 + Adjust the Column ordering.
 
+|Parameter|Description|
+|---------|-----------|
+|**Created**| When the log was created mm/dd/yyyy|
+|**Provider**|List of your providers|
+|**MOS- Mean Opinion Score**|To find the call quality|
+|**DTMF- Dual Tone Multi Frequency**|diagnose and resolve various communication problems, enhancing network performance and ensuring smooth communication experiences|
+|**Answered**|Call has been connected and answered|
+|**Destroyed**|Call has been killed|
+|**Direction**|Either **Termination** or **Origination**|
```

Comment:
Specifies whether the call is Termination or Origination.

Reply:
done